nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -706,11 +708,18 @@ in
     };
 
     systemd.services.nginx-config-reload = mkIf cfg.enableReload {
-      wantedBy = [ "nginx.service" ];
+      wants = [ "nginx.service" ];
+      wantedBy = [ "multi-user.target" ];
       restartTriggers = [ configFile ];
+      # commented, because can cause extra delays during activate for this config:
+      #      services.nginx.virtualHosts."_".locations."/".proxyPass = "http://blabla:3000";
+      # stopIfChanged = false;
+      serviceConfig.Type = "oneshot";
+      serviceConfig.TimeoutSec = 60;
       script = ''
         if ${pkgs.systemd}/bin/systemctl -q is-active nginx.service ; then
-          ${pkgs.systemd}/bin/systemctl reload nginx.service
+          ${execCommand} -t && \
+            ${pkgs.systemd}/bin/systemctl reload nginx.service
         fi
       '';
```

Comment:
This line breaks nginx. It causes directories in `/var/cache/nginx/` to be owned by `nobody`, which breaks large POST uploads. I think this might due the fact that nginx is no longer started as root. cc @Izorkin

Reply:
#85820 - fix variant and https://github.com/NixOS/nixpkgs/pull/85820#issuecomment-620378045

Reply:
Ok. I have a simpler version. Will post.

Reply:
https://github.com/NixOS/nixpkgs/pull/95249